AutonomousDemo/VehicleMotion.cs

```diff
@@ -10,24 +10,34 @@ public class VehicleMotion
     {
```

Comment:
Originally, more generally.  Just because that's how it made sense to me starting out.